runelite-client/src/main/java/net/runelite/client/plugins/timers/TimersPlugin.java

```diff
@@ -325,6 +354,15 @@ public void onConfigChanged(ConfigChanged event)
 	@Subscribe
 	public void onMenuOptionClicked(MenuOptionClicked event)
 	{
+		if (config.showAntiPoison()	&& event.getMenuOption().contains("Cure"))
+		{
+			// what was drunk? we don't know.
+			// so, take inv snapshot before your character takes a sip
+			this.prevItems = client.getItemContainer(InventoryID.INVENTORY).getItems().clone();
```

Comment:
`this` isn't needed; most of the project avoids its usage unless explicitly needed (in constructors for instance)
```suggestion
			prevItems = client.getItemContainer(InventoryID.INVENTORY).getItems().clone();
```